### PR TITLE
cli cleanup

### PR DESCRIPTION
**Pull request by yasminosb**
all indentation should be flawless now, no more trailing newlines caused by the default paramaters that were not considered yet

### CODE REVIEW

File: src/lib/options.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export interface CustomOptions {
   maxGapSize?: number;
   maxHashCount?: number;
   maxHashPercent?: number;
-  maxMatches?: number;
+  maxMatches?: number; // still needed?
```
Comment (rien) [inline src/lib/options.ts]:
```suggestion
  maxMatches?: number;
```

The `maxMatches` option is still used [here](https://github.com/dodona-edu/dolos/blob/1d6061497c34918555f7e4b02b54f9a53c3bb76a/src/lib/summary.ts#L89), or what do you mean?

File: src/cli.ts [hunk 2]
```diff
@@ -125,7 +139,7 @@ program
         maxGapSize: program.maxGapSize,
         maxHashCount: program.maximumHashCount,
         maxHashPercent: program.maximumHashPercentage,
-        maxMatches: program.filePairOutputLimit,
+        maxMatches: program.filePairOutputLimit, // still needed?
```
Comment (rien) [inline src/cli.ts]:
```suggestion
        maxMatches: program.filePairOutputLimit,
```

File: src/lib/options.ts [hunk 3]
```diff
@@ -89,7 +89,7 @@ export class Options {
     return definedOrDefault(this.custom.clusterMinMatches, Options.defaultClusterMinMatches);
   }
 
-  get maxMatches(): number | null {
+  get maxMatches(): number | null { // still needed?
```
Comment (rien) [inline src/lib/options.ts]:
```suggestion
  get maxMatches(): number | null {
```

File: src/lib/options.ts [hunk 4]
```diff
@@ -105,7 +105,7 @@ export class Options {
       maxGapSize: this.maxGapSize,
       maxHashCount: this.custom.maxHashCount,
       maxHashPercent: this.filterByPercentage ? this.maxHashPercent : undefined,
-      maxMatches: this.maxMatches,
+      maxMatches: this.maxMatches, // still needed?
```
Comment (rien) [inline src/lib/options.ts]:
```suggestion
      maxMatches: this.maxMatches,
```

File: src/lib/options.ts (hunk 1 above)
Comment (yasminosb) [reply src/lib/options.ts]: I must have overlooked that reference! I removed the comments.

**Comment by rien** [comment]
Merged manually with aed2aa1

**Pull request closed without merging**